### PR TITLE
Configure PHP version to install

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: infrastructure/docker/services/php-base/Dockerfile [hunk 1]
```diff
@@ -13,13 +13,15 @@ RUN apt-get update \
...
+ARG PHP_VERSION
+
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         php${PHP_VERSION}-apcu \
+        php${PHP_VERSION}-bcmath \
+        php${PHP_VERSION}-cli \
         php${PHP_VERSION}-common \
         php${PHP_VERSION}-curl \
-        php${PHP_VERSION}-iconv \
```
Comment (lyrixx) [inline infrastructure/docker/services/php-base/Dockerfile:22]: Why do you remove this packages?

**Review by lyrixx — COMMENTED** [review]
Is is PR ready?

File: infrastructure/docker/services/php-base/Dockerfile (hunk 1 above)
Comment (pyrech) [reply infrastructure/docker/services/php-base/Dockerfile:22]: Because it failed in some PHP version, but I can remembe which one (probably 7.2)

Comment (lyrixx) [reply infrastructure/docker/services/php-base/Dockerfile:22]: IMHO, it's better to keep it, and people that use 7.2 will have to remove it manually.
https://www.php.net/supported-versions.php => PHP 7.2 is not supported anymore (security fix only until end of 2020). So I don't really care if the docker started does not work out of the box with unsupporte PHP version :)

Comment (pyrech) [reply infrastructure/docker/services/php-base/Dockerfile:22]: I just tested with this package required and it everything worked fine on PHP versions from 7.2 to 7.4, so let's keep it as it is \o/

**Comment by pyrech** [comment]
PR ready for merge

**Review by lyrixx — APPROVED** [review]
👍🏼 thx

**Pull request merged**